### ISSUE TITLE
Kick on joining

### ISSUE THREAD

**Issue by josua27176** [VelocityPowered/Velocity#318]
Basically we have this one player that every time he joins it kicks only him and 2 other players at random and then throws all these errors in the console log.
```
[19:11:48] [Netty epoll Worker #1/ERROR]: [server connection] Josua27176 -> pixel: exception encountered in com.velocitypowered.proxy.connection.backend.BackendPlaySessionHandler@22ebfecd
java.lang.IllegalStateException: java.io.UTFDataFormatException: malformed input around byte 17
	at com.velocitypowered.proxy.protocol.util.ByteBufDataInput.readUTF(ByteBufDataInput.java:107) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at com.velocitypowered.proxy.connection.backend.BungeeCordMessageResponder.processForwardToServer(BungeeCordMessageResponder.java:224) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at com.velocitypowered.proxy.connection.backend.BungeeCordMessageResponder.process(BungeeCordMessageResponder.java:292) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at com.velocitypowered.proxy.connection.backend.BackendPlaySessionHandler.handle(BackendPlaySessionHandler.java:101) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at com.velocitypowered.proxy.protocol.packet.PluginMessage.handle(PluginMessage.java:81) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at com.velocitypowered.proxy.connection.MinecraftConnection.channelRead(MinecraftConnection.java:113) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.handler.flow.FlowControlHandler.dequeue(FlowControlHandler.java:191) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.handler.flow.FlowControlHandler.channelRead(FlowControlHandler.java:153) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:324) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:296) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:286) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.epoll.AbstractEpollStreamChannel$EpollStreamUnsafe.epollInReady(AbstractEpollStreamChannel.java:792) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:475) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:378) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_252]
Caused by: java.io.UTFDataFormatException: malformed input around byte 17
	at java.io.DataInputStream.readUTF(DataInputStream.java:656) ~[?:1.8.0_252]
	at com.velocitypowered.proxy.protocol.util.ByteBufDataInput.readUTF(ByteBufDataInput.java:105) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	... 41 more
[19:11:48] [Netty epoll Worker #1/INFO]: [connected player] Josua27176 (/24.34.161.59:43539) has disconnected: Your connection to pixel encountered an error.
[19:11:48] [Netty epoll Worker #1/INFO]: [server connection] Josua27176 -> pixel has disconnected
[19:11:48] [Velocity Event Executor - #0/INFO]: Josua27176 has left the proxy
[19:11:50] [Netty epoll Worker #1/ERROR]: [server connection] Kobrakill00 -> pixel: exception encountered in com.velocitypowered.proxy.connection.backend.BackendPlaySessionHandler@735ee552
java.lang.IllegalStateException: java.io.UTFDataFormatException: malformed input around byte 17
	at com.velocitypowered.proxy.protocol.util.ByteBufDataInput.readUTF(ByteBufDataInput.java:107) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at com.velocitypowered.proxy.connection.backend.BungeeCordMessageResponder.processForwardToServer(BungeeCordMessageResponder.java:224) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at com.velocitypowered.proxy.connection.backend.BungeeCordMessageResponder.process(BungeeCordMessageResponder.java:292) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at com.velocitypowered.proxy.connection.backend.BackendPlaySessionHandler.handle(BackendPlaySessionHandler.java:101) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at com.velocitypowered.proxy.protocol.packet.PluginMessage.handle(PluginMessage.java:81) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at com.velocitypowered.proxy.connection.MinecraftConnection.channelRead(MinecraftConnection.java:113) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.handler.flow.FlowControlHandler.dequeue(FlowControlHandler.java:191) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.handler.flow.FlowControlHandler.channelRead(FlowControlHandler.java:153) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:324) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:311) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:425) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:276) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:286) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.epoll.AbstractEpollStreamChannel$EpollStreamUnsafe.epollInReady(AbstractEpollStreamChannel.java:792) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:475) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:378) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_252]
Caused by: java.io.UTFDataFormatException: malformed input around byte 17
	at java.io.DataInputStream.readUTF(DataInputStream.java:656) ~[?:1.8.0_252]
	at com.velocitypowered.proxy.protocol.util.ByteBufDataInput.readUTF(ByteBufDataInput.java:105) ~[velocity-proxy-1.1.0-SNAPSHOT-all.jar:1.1.0-SNAPSHOT (git-e83662e8-b117)]
	... 43 more
[19:11:50] [Netty epoll Worker #1/INFO]: [connected player] Kobrakill00 (/184.188.64.175:58698) has disconnected: Your connection to pixel encountered an error.
[19:11:50] [Netty epoll Worker #1/INFO]: [server connection] Kobrakill00 -> pixel has disconnected
[19:11:50] [Velocity Event Executor - #0/INFO]: Kobrakill00 has left the proxy
```

**Comment by josua27176** [comment]
I am just looking for a solution also for some reason our proxy pings VERY VERY SLOWLY. And like i have put tcp-fast on, Ive tried disabling the ping passthrough everything.

**Comment by astei** [comment]
You have a backend plugin that is sending malformed BungeeCord plugin messages, but it's also likely there is a bug in Velocity itself. Let's try to isolate the issue.

Can you try with the following proxy configurations?

* Velocity 1.0.7 with BungeeQuack plugin
* Waterfall

> also for some reason our proxy pings VERY VERY SLOWLY

This looks like a separate issue. To me, this sounds like a network problem, a plugin taking too long to send ping responses, or the Minecraft client simply having problems pinging servers.

**Comment by josua27176** [comment]
Were currently running a Pixelmon server using CatServer on 1.12.2. I will run both of those configurations now and get back. As for the proxy pinging issue would that be a velocity plugin or like a per server plugin because it happens even on servers that arent up.

**Comment by astei** [comment]
> because it happens even on servers that arent up.

Explain more. If we're talking about servers in the client server list that aren't up, then it's clearly a client problem. Velocity isn't at fault here.

**Comment by josua27176** [comment]
> Explain more. If we're talking about servers in the client server list that aren't up, then it's clearly a client problem. Velocity isn't at fault here.

Sorry I what I mean is you know how Velocity does the ping passthrough and forced host even if the host is down that is being forced it still shows it delayed so I feel like it may be a proxy plugin if you think its that.

**Comment by josua27176** [comment]
Update:
When I tried just a plan 1.0.7 Velocity the ping issue was resolved we just gotta confirm to see if its resolved with the player who joins.

**Comment by astei** [comment]
Velocity 1.0.7 doesn't have ping passthrough so I wonder if the problem is with that.

**Comment by astei** [comment]
I still want us to focus on the BungeeCord plugin message channel bug though.

**Comment by josua27176** [comment]
Ok thats fine we can work on that

**Comment by josua27176** [comment]
Well we are using Catalyst for our Velocity and it enables ping pass-through and stuff.

**Comment by josua27176** [comment]
But I am waiting for the guy to get back he is afk right now

**Comment by josua27176** [comment]
Update: He is able to join WITHOUT issue and all ping related issues seem to be resolved but now we cant have the custom stuff we want like the per server MOTDs and stuff.

**Comment by astei** [comment]
So Velocity 1.0.7 with BungeeQuack works, now we need to figure out why 1.1.0 doesn't like it. I'll investigate further when I get time.

**Comment by josua27176** [comment]
Ok are we talking like tomorrow or like a week from now? Also Ill keep trying different combinations and seeing what it likes and what it doesnt.

**Comment by astei** [comment]
As a normal practice I don't give ETAs. This is an open source project and I'm not obligated to provide anyone with specific timeframes. All I can tell you is that I will be investigating why Velocity 1.1.0's native BungeeCord forwarding isn't working and how I can fix it.

**Comment by astei** [comment]
This should be fixed with d6b1bc2. I carefully looked at how BungeeCord read the message and noticed we were not reading the message correctly. This should be fixed now.
